services/login/src/handlers/mozilla-auth0.js

```diff
@@ -80,12 +76,30 @@ class Handler {
     return this._managementApi;
   }
 
-  async profileFromUserId(userId) {
+  async getUser({userId, checkIfActive}) {
     const a0 = await this.getManagementApi();
+    const userProfile = await a0.getUser({id: userId});
+
+    if (checkIfActive && 'active' in userProfile && !userProfile.active) {
+      debug('user is not active; rejecting');
+      return;
+    }
+
+    const user = new User();
+
+    user.identity = this.identityFromProfile(userProfile);
+    if (!user.identity) {
+      debug('No recognized identity providers');
+      return;
+    }
 
-    return a0.getUser({id: userId});
+    // take a user and attach roles to it
+    this.addRoles(userProfile, user);
+
+    return user;
   }
 
+  // exposed method
```

Comment:
It would probably be good to document the Handler API in a README or maybe in a parent class.  But again, maybe not useful if we'll be rewriting all of this code into services/web-server soon, maybe even with a totally different "handler" API.

Reply:
If this is going to be rewritten soon - I'd just leave these in the code as comments